server/routes/participant-user.js

```diff
@@ -163,7 +163,7 @@ router.post(
         res.status(422).send('Already Hired or Withdrawn');
       }
     } else {
-      res.status(422).send(`No expression of interest with id: ${id}`);
+      res.status(422).send(`No expression of interest with participant id`);
```

Comment:
Can we rephrase this error message a bit maybe? It sounds quite ambiguous

Reply:
what do you suggest?

Reply:
There's also the same issue on line 135

Reply:
Maybe something like _No expression of interest found for this participant_?